README.md

```diff
@@ -1 +1,68 @@
-# webpack-broilerplate
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# LeaderBoard
+
+> A JavaScript project for the Leaderboard list app, using webpack and ES6 features, this is a requirement for Micronauts in Module 2 block 4. This project is MIT Licensed
+
+## Built With
+
+- HTML and CSS
+- Javascript ES6
+- Webpack
+- HtmlWebpackPlugin
+- GitFlow
+
+## How to use and run this project
+
+> To get a local copy up and running follow these simple example steps.
+
+-Clone this repository with
+
+<code> git clone git@github.com:aceDavon/LeaderBoard.git </code>
```

Comment:
- [ ] I think, here should be your repo name. Actually, it is `aceDavon`.

Reply:
Clone repo in readme fixed